docs/en/table_design/Data_distribution.md

```diff
@@ -199,166 +200,327 @@ You only need to configure a partition expression (a time function expression or
 
 #### Range partitioning
 
-Range partitioning is suitable for storing simple, contiguous data, such as time series data (dates or timestamps), or continuous numerical data. And you frequently query and manage data based on continuous date/numerical ranges. Also, it can be applied in some special cases where historical data needs to be partitioned by month, and recent data needs to be partitioned by day.
+Range partitioning is suitable for storing simple, contiguous data, such as time series data, or continuous numerical data. And you frequently query and manage data based on continuous date/numerical ranges. Also, it can be applied in some special cases where historical data needs to be partitioned by month, and recent data needs to be partitioned by day.
 
-StarRocks stores data in the corresponding partitions based on the explicit mapping of the explicitly defined range for each partition.
+You need to explicitly define the data partitioning columns and establish the mapping relationship between partitions and ranges of partitioning column values. During data loading, StarRocks assigns the data to the corresponding partitions based on the ranges to which the data partitioning column values belong.
 
-##### Dynamic partitioning
+As for the data type of the partitioning columns, before v3.3.0, range partitioning only supports partitioning columns of date and integer types. Since v3.3.0, three specific time functions to be used as partition columns. When explicitly defining the mapping relationship between partitions and ranges of partitioning column values, you need to first use a specific time function to convert partitioning column values of timestamps or strings into dates, and then divide the partitions based on the converted dates.
 
-[Dynamic partitioning](./dynamic_partitioning.md) related properties are configured at table creation. StarRocks automatically creates new partitions in advance and removes expired partitions to ensure data freshness, which implements time-to-live (TTL) management for partitions.
+:::info
 
-Different from the automatic partition creation ability provided by the expression partitioning, dynamic partitioning can only periodically create new partitions based on the properties. If the new data does not belong to these partitions, an error is returned for the load job. However, the automatic partition creation ability provided by the expression partitioning can always create corresponding new partitions based on the loaded data.
+- If the partitioning column value is a timestamp, you need to use the from_unixtime or from_unixtime_ms function to convert the timestamp to the date when dividing the partitions. When the from_unixtime function is used, the partitioning column only supports INT and BIGINT types. When using the from_unixtime_ms function is used, the partitioning column only supports BIGINT type.
```

Comment:
When using the from_unixtime_ms function is used, --- When the from_unixtime_ms function is used,